docs/kafka/quarkus-kafka/README.adoc

```diff
@@ -177,13 +177,13 @@ endif::[]
 == Creating a Kafka topic in {product-kafka}
 
 [role="_abstract"]
-For this quick start, the Kafka topic that the Quarkus example application references is called `prices`. You need to create this topic in {product-long-kafka} so that the Quarkus application can interact with it.
+The Quarkus application in this quick start uses a Kafka topic called `prices` to produce and consume messages. In this task, you'll create the topic in your Kafka instance.
```

Comment:
Suggest slight rewrite of  'In this task, you'll create the 'prices' topic in your Kafka instance.' just to remove any ambiguity.